.github/workflows/cypress-workflow-bundle-snapshot-based.yml

```diff
@@ -17,12 +17,12 @@ jobs:
     with:
       test-name: Core Dashboards using Bundle Snapshot
       test-command: env CYPRESS_NO_COMMAND_LOG=1 CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-with-security --browser chromium --spec 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js'
-      osd-serve-args: --data_source.enabled=true --vis_builder.enabled=true --ml_commons_dashboards.enabled=true
+      osd-serve-args: --data_source.enabled=true --data_source.ssl.verificationMode=none --vis_builder.enabled=true --ml_commons_dashboards.enabled=true 
```

Comment:
just a heads up.

the cypress workflow in OSD doesn't run these tests per PR. It runs with the [ciGroup scripts](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/package.json#L28).

The workflow then calls them here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.github/workflows/cypress_workflow.yml#L122

Then OSD gets started up with:
https://github.com/opensearch-project/OpenSearch-Dashboards/blame/main/.github/workflows/cypress_workflow.yml#L33

Should creating a new start cmd in OSD? that does something like `yarn start --data_source.enabled --data_source.ssl.verificationMode=none` Then create a new ciGroup(s) for MDS? To start up OSD with MDS and  calls the script from this repo to get the MDS tests and run them per PR.

Reply:
@kavilla Thanks for sharing how cypress workflow in OSD runs tests in functional test repo! I think we can just modify the existing  `START_CMD` to add `--data_source.enabled=true, data_source.ssl.verificationMode=none`. For ciGroup, I think there's an existing one for MDS https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/79d98cabc16f43a781b8638dcad1e34d5951390d/package.json#L32